tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectLoadingTest.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
```

Comment:
This file is called `TiledMapObjectLoadingTest.java` but it contains a class named `TiledMapDirectLoaderTest`. Does this even compile? I looks like a copy paste mistake.